types/events.go

```diff
@@ -92,6 +86,10 @@ type EventDataVote struct {
 
 type EventDataString string
 
+type EventDataValidatorSetUpdates struct {
+	ValidatorUpdates []abci.ValidatorUpdate `json:"validator_updates"`
```

Comment:
is it weird that we're going to be passing an ABCI validator over the RPC here ? ie. it will be different than the usual representation of pubkeys using AminoJSON